proofwatch/evidence/evidence.go

```diff
@@ -0,0 +1,46 @@
+package evidence
+
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/in-toto/go-witness/cryptoutil"
+)
+
+// EvidenceEvent represents a higher-level, mapped conformance assertion.
+type EvidenceEvent struct {
+	Summary   string      `json:"summary"`
+	Timestamp time.Time   `json:"timestamp"`
+	Evidence  RawEvidence `json:"evidence"`
+}
+
+func NewFromEvidence(rawEnv RawEvidence) *EvidenceEvent {
+	event := EvidenceEvent{
+		Timestamp: time.Now(),
+		Evidence:  rawEnv,
+	}
+	return &event
+}
+
+// RawEvidence represents a simplified raw output from a policy engine.
+type RawEvidence struct {
+	Metadata `json:,inline`
```

Comment:
`inline` seems to be invalid here. Maybe using simply `Metadata` or

> Metadata `json:"metadata"` ?

Depends on how would be the desired final state, nested or flatten...